src/PackageCompilerX.jl

```diff
@@ -19,18 +34,28 @@ all_stdlibs() = readdir(Sys.STDLIB)
 yesno(b::Bool) = b ? "yes" : "no"
 
 function bitflag()
-    if Sys.ARCH == :aarch64 || Sys.ARCH == :arm
-        return ``
+    if Sys.ARCH == :i686
+        return `-m32`
+    elseif Sys.ARCH == :x86_64
+        return `-m64`
     else
-        return Int == Int32 ? `-m32` : `-m64`
+        return ``
     end
 end
 
 function march()
-    if Sys.ARCH == :aarch64 || Sys.ARCH == :arm
-        return (Int == Int32 ? `-march=armv7-a` : `-march=armv8-a+crypto+simd`)
+    if Sys.ARCH === :i686
+        return "-march=pentium4"
+    elseif Sys.ARCH === :x86_64
+        return "-march=x86-64"
+    elseif Sys.ARCH === :arm
+        return "-march=armv7-a+simd"
```

Comment:
- I could confirm `-march=armv7-a+simd` works fine for building `HelloX.jl` on my Raspberry Pi locally.

Reply:
Thanks!

Reply:
So if we use `+simd`, you can see in [the `-march=name` table on this page](https://gcc.gnu.org/onlinedocs/gcc/ARM-Options.html) we enable VFPv3 and NEON.  If you look at [the Raspberry PI FPU table on wikipedia](https://en.wikipedia.org/wiki/Raspberry_Pi#Specifications), and we only pay attention to the Model B, you'll see that is sufficient for the 2B and onward.  IMO, that's good enough, so let's just use this.  It's going to be really painful to use Julia without those options, and we don't have great BB support for manipulating these flags, so this should be a realistically conservative set of options.